src/StreamingMiddleware.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Amp\Http\Server\FormParser;
+
+use Amp\Http\Server\Middleware;
+use Amp\Http\Server\Request;
+use Amp\Http\Server\RequestHandler;
+use Amp\Promise;
+
+class StreamingMiddleware implements Middleware
```

Comment:
I don't think this middleware makes sense. It's a single line that's replaced with another line if this middleware is used. And the use cases for streamed body parsing are rather rare.

Reply:
Yeah, making both made sense at the time, but this middleware doesn't do enough. I'll remove it.